test/Sentry.Maui.Tests/SentryMauiLogcatsTests.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.Extensions.Options;
+#if ANDROID
+using Sentry.Android;
+#endif
+
+namespace Sentry.Maui.Tests;
+
+public class SentryMauiLogcatsTests
+{
+    private class Fixture
+    {
+        public MauiAppBuilder Builder { get; }
+        public FakeTransport Transport { get; private set; } = new FakeTransport();
+        public InMemoryDiagnosticLogger Logger { get; private set; } = new InMemoryDiagnosticLogger();
+
+        public Fixture()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder.Services.AddSingleton(Substitute.For<IApplication>());
+
+            builder.Services.Configure<SentryMauiOptions>(options =>
+            {
+                options.Transport = Transport;
+                options.Dsn = ValidDsn;
+                options.AttachScreenshot = false; //Disable the screenshot attachment to have the logcat as primary attachment
+                options.Debug = true;
+                options.DiagnosticLogger = Logger;
+                options.AutoSessionTracking = false; //Get rid of session envelope for easier Assert
+                options.CacheDirectoryPath = null;   //Do not wrap our FakeTransport with a caching transport
+                options.FlushTimeout = TimeSpan.FromSeconds(10);
+            });
+            Builder = builder;
+        }
+    }
+
+    private readonly Fixture _fixture = new();
+
+#if __MOBILE__
+    [SkippableFact]
```

Comment:
```suggestion
#if ANDROID
    [Fact]
```

The test is only relevant on ANDROID right? We could simplify this and get rid of a bunch of the other directive checks below...

Reply:
it makes more sense yes, now it might be there is gonna be a bit of a conflict with the other PR but nothing we can't handle